### PR TITLE
Add warning about GUI deprecation

### PR DESCRIPTION
**Pull request by t-bast**
The GUI has been deprecated a long time ago, and doesn't inform users about potential risks, such as RBF-ing funding txs.

We should instead incentivize users to use the CLI and read the documentation, which ensures they will know about potential pitfalls. I have added a warning about RBF to the wiki documentation [here](https://github.com/ACINQ/eclair/wiki/Usage).

This is related to #1926

### CODE REVIEW

**Comment by Kixunil** [comment]
We also had a bunch of issues in `electrs` because of people reading outdated/low-quality guides, although thankfully not as catastrophic as #1926. We ended up adding big warnings to several places saying people shouldn't blindly follow other guides than our own. I can't say how effective it is because we only added it a few days ago but I think adding such warning here shouldn't hurt.

**Comment by t-bast** [comment]
Good idea, I'll add that!

File: README.md [hunk 1]
```diff
@@ -97,6 +98,12 @@ eclair-node-<version>-<commit_id>/bin/eclair-node.sh
 
 You can then control your node via the [eclair-cli](https://github.com/ACINQ/eclair/wiki/Usage) or the [API](https://github.com/ACINQ/eclair/wiki/API).
 
+This repository also contains a sample GUI ([eclair-node-gui](https://github.com/ACINQ/eclair/tree/master/eclair-node-gui)).
+
+:warning: The sample GUI should only be used for prototyping and demo purposes: it is unsafe to use it on mainnet.
```
Comment (pm47) [inline README.md:103]:
```suggestion
:rotating_light: The sample GUI should only be used for prototyping and demo purposes: using it on mainnet is unsafe.
```

**Review by pm47 — COMMENTED** [review]
How about completely dropping the gui in eclair 0.7?

**Comment by t-bast** [comment]
> How about completely dropping the gui in eclair 0.7?

I'm down with that! Kill the old witch!
Then I'll only keep the warning about obsolete guides in this PR.

File: README.md (hunk 1 above)
Comment (t-bast) [reply README.md]: Done in https://github.com/ACINQ/eclair/pull/1929/commits/7474d3ce32e45f162e7b82ada0ab05396158eb9e

**Review by pm47 — APPROVED** [review]

**Pull request merged**